projects/nano-editor.org/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.major }}.{{ version.minor }}.tar.gz
+
+versions:
+  - 7.2
+
+dependencies:
+  gnu.org/gettext: "*"
+  invisible-island.net/ncurses: ">=6.0"
+
+provides:
+  - bin/nano
+
+build:
+  dependencies:
+    freedesktop.org/pkg-config: "*"
+    gnu.org/automake: "*"
+    gnu.org/make: "*"
+    tea.xyz/gx/cc: "*"
+  script: |
+    cd nano-$VERSION && ./configure $ARGS
+    make install
+  env:
+    VERSION: "{{ version.major }}.{{ version.minor }}"
```

Comment:
I believe this can be:
```yaml
script: |
  cd nano-{{ version.marketing }}
  ./configure $ARGS
  make install
```

You can _try_:
```yaml
working-directory: nano-{{ version.marketing }}
```
instead, but I don't know if that passes through the de-moustaching.

Reply:
that made it nicer.